exercises/branching-conditionals/rockPaperScissorsWinner/rockPaperScissorsWinner.js

```diff
@@ -24,15 +24,49 @@
  * @returns {string} Returns 'win', 'lose', or 'draw' depending on
  *  whether the first player won, lost, or drew the match.
  */
-function rockPaperScissorsWinner(array) {
+function rockPaperScissorsWinner(firstPlayerMove, secondPlayerMove) {
```

Comment:
Look at the value of `firstPlayerMove - secondPlayerMove % 3` and see if you can't use that to simplify the nest of conditionals below.